### PR TITLE
dev-util/nvidia-cuda-toolkit: Bump version to 10.1.243

### PR DESCRIPTION
**Pull request by jvesely**
Adapt to the new download location.
Adapt to new directory names for nsight-compute and nsight-systems.
Use rm -r instead of rm -rf to catch removing non-existent files.
Bump disk requirements to 6.8GB (cuda is now 3.4GB unpacked).

Signed-off-by: Jan Vesely <jano.vesely@gmail.com>

### CODE REVIEW

**Comment by jvesely** [comment]
@perfinion, @amadio, does this look ok to you?

**Comment by jvesely** [comment]
I've tested the new version by compiling/running a small example.cu file and running a pycuda example. Both worked ok.

**Comment by jvesely** [comment]
@perfinion, does this one look ok to you?

**Comment by jvesely** [comment]
ping

**Comment by jvesely** [comment]
ping

File: dev-util/nvidia-cuda-toolkit/nvidia-cuda-toolkit-10.1.243.ebuild [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+DESCRIPTION="NVIDIA CUDA Toolkit (compiler and friends)"
+HOMEPAGE="https://developer.nvidia.com/cuda-zone"
+SRC_URI="https://developer.download.nvidia.com/compute/cuda/${MYD}/Prod/local_installers/cuda_${PV}_${DRIVER_PV}_linux.run -> cuda_${PV}_${DRIVER_PV}_linux.run"
+
+LICENSE="NVIDIA-CUDA"
+SLOT="0/${PV}"
+KEYWORDS="-* ~amd64 ~amd64-linux"
+IUSE="debugger doc eclipse profiler"
+
+DEPEND=""
```
Comment (alexxy) [inline dev-util/nvidia-cuda-toolkit/nvidia-cuda-toolkit-10.1.243.ebuild]: should be BDEPEND

**Comment by jvesely** [comment]
ping.
 @alexxy., any thoughts on the new version?

**Comment by jvesely** [comment]
ping

**Comment by perfinion** [comment]
Thanks for this. sorry i didnt get to it earlier. I made some tiny changes (it needs the openssl-compat dep now too)

**Comment by GBeauregard** [comment]
> Thanks for this. sorry i didnt get to it earlier. I made some tiny changes (it needs the openssl-compat dep now too)

There's a gentoo bug for this [https://bugs.gentoo.org/690998](url), it affects the older ebuilds too.

**Pull request closed without merging**